src/commands/upgrade.ts

```diff
@@ -85,8 +88,8 @@ export class UpgradeCommand extends Command {
             .setTitle('Upgrade card')
             .setThumbnail('attachment://card.png')
             .setDescription([
-                `Upgrading the \`${card.mapper.username}\` card from from **${condition.id}** to **${condition.nextUpgrade.id}** has a ${Math.floor(condition.upgradeChance * 100)}% chance of success.`,
-                condition.previousUpgrade ? `If the upgrade fails, there is a 50% chance the card will be downgraded to **${condition.previousUpgrade.id}**` : `If the upgrade fails, nothing will happen.`,
+                `Upgrading the \`${card.mapper.username}\` card from from **${condition.id}** to **${condition.nextUpgrade.id}** has a ${Math.floor(upgradeChance * 100)}% chance of success.`,
+                condition.previousUpgrade ? `If the upgrade fails, there is a ${Math.floor(downgradeChance * 100)}% chance the card will be downgraded to **${condition.previousUpgrade.id}**` : `If the upgrade fails, nothing will happen.`,
```

Comment:
I don't think the pity factor should be advertised anywhere

Reply:
it displays the odds and the odds change, felt like basic transparency

Reply:
I'd still prefer not to give it away, ppl should not really be aware of the pity system